javasource/audittrail/log/CreateLogObject.java

```diff
@@ -144,9 +144,11 @@ public static IMendixObject createAuditLogItems(final IMendixObject auditableObj
 								&& (ass.getType() == AssociationType.REFERENCESET
 										|| ass.getOwner() != AssociationOwner.BOTH)) {
 							association = ass.getName();
-							logObject.setValue(sudoContext, association, auditableObject.getId());
-
-							setAssociationName(auditableObject.getType(), association);
+							if(!ass.getName().equals(Log.MemberNames.Log_User.toString())) {

```

Comment:
We should merge this condition with its parent condition and move the `association = ass.getName();` line inside the same block as the `.setValue` and `setAssociationName`.

This will require a small change to our own mpr, which is incorrect and is missing the second association to `System.User`.